common/scripted_effects/wc_scheme_effects.txt

```diff
@@ -0,0 +1,238 @@
+﻿#Sets up the Outcome Roll values
+stasis_trap_outcome_roll_setup_effect = {
```

Comment:
Is it me or every scheme has this effect unchanged? So there's no way to have it? You can just make one universal effect.